charts/azimuth-llm/values.yaml

```diff
@@ -104,7 +103,7 @@ ui:
   # Container image config
   image:
     repository: ghcr.io/stackhpc/azimuth-llm-chat-ui
-    tag:  # Defaults to chart's appVersion
+    tag: # Defaults to chart's appVersion
```

Comment:
Note: Helm chart linter will complain about too few spaces before comment - probably need to correct this as part of version bump workflow